merlin-server/build.gradle

```diff
@@ -42,6 +42,21 @@ task generateDocumentation(type: NpmTask) {
   inputs.dir('src')
 }
 
+task cleanDocsFromDocsDir(type: Delete) {
+  delete rootDir.toPath().resolve('docs/edsl-api-docs/constraints-edsl-api/develop')
```

Comment:
Is this path the part that will need to be updated in order to support multiversioning on the EDSL docs?

Reply:
So we decided that as part of the release of a new version, we take the existing develop and move it to a versioned dir that gets committed. So in that way, we always have develop building, and the versioned ones we just handle manually for now. (typedoc doesn't have the cool git integration sphinx does)